README.rst

```diff
@@ -33,28 +33,34 @@ Get started with tutorials
 
 * (Easiest) Improve a simple classifier from 60% to 80% accuracy on the Iris dataset:
   
-  * `TUTORIAL: simple cleanlab on Iris <https://github.com/cleanlab/examples/blob/master/iris_simple_example.ipynb>`__.
+  * `TUTORIAL: simple cleanlab on Iris <https://github.com/cleanlab/examples/blob/master/iris_simple_example.ipynb>`__
+
+* (Comprehensive) Image classification with noisy labels
+
+  * `TUTORIAL: learning with noisy labels on CIFAR <https://github.com/cleanlab/examples/tree/master/cifar10>`__
 
 * Run Cleanlab on 4 datasets using 9 different classifiers/models:
   
-  * `TUTORIAL: classifier comparison <https://github.com/cleanlab/examples/blob/master/classifier_comparison.ipynb>`__.
+  * `TUTORIAL: classifier comparison <https://github.com/cleanlab/examples/blob/master/classifier_comparison.ipynb>`__
 
-* Find `label errors <https://openreview.net/forum?id=XccDXrDNLek>`_ in MNIST, ImageNet, CIFAR-10/100, Caltech-256, QuickDraw, Amazon Reviews, IMDB, 20 Newsgroups, & AudioSet:
+* Find `label errors <https://arxiv.org/abs/2103.14749>`_ in MNIST, ImageNet, CIFAR-10/100, Caltech-256, QuickDraw, Amazon Reviews, IMDB, 20 Newsgroups, AudioSet:
 
-  * `TUTORIAL: Find Label Errors in the 10 most common ML benchmark test datasets with Cleanlab <https://github.com/cleanlab/label-errors/blob/main/examples/Tutorial%20-%20How%20To%20Find%20Label%20Errors%20With%20CleanLab.ipynb>`__.
+  * `TUTORIAL: Find Label Errors in the 10 most common ML benchmark test datasets with cleanlab <https://github.com/cleanlab/label-errors/blob/main/examples/Tutorial%20-%20How%20To%20Find%20Label%20Errors%20With%20CleanLab.ipynb>`__
 
-* Demystifying `Confident Learning <https://www.jair.org/index.php/jair/article/view/12125>`_ with just numpy and for-loops:
```

Comment:
Dropped "with just numpy and for-loops" because it says this same phrase right below in the name of the tutorial. Also now added the "visualizing confident learning" tutorial under this Demystifying header for which this "with just numpy and for-loops" phrase does not apply.